### PR TITLE
zh-CN: Synchronize the translation of createAnalyser

### PR DESCRIPTION
**Pull request by Yukiniro**
en-US: https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createAnalyser
zh-CN: https://developer.mozilla.org/zh-CN/docs/Web/API/BaseAudioContext/createAnalyser

### CODE REVIEW

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md [hunk 1]
```diff
@@ -83,6 +87,6 @@ function draw() {
 
 {{Compat}}
 
-## 另见
+## 参见
 
-- [Using the Web Audio API](/zh-CN/docs/Web_Audio_API/Using_Web_Audio_API)
+- [Using the Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API/Using_Web_Audio_API)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:92]:
```suggestion
- [使用 Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API/Using_Web_Audio_API)
```

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md [hunk 2]
```diff
@@ -1,78 +1,82 @@
 ---
-title: AudioContext.createAnalyser()
+title: "BaseAudioContext: createAnalyser() 方法"
```
Comment (jasonren0403) [inline files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:2]:
```suggestion
title: "BaseAudioContext：createAnalyser() 方法"
```

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md [hunk 3]
```diff
@@ -1,78 +1,82 @@
...
 ---
 
-{{ APIRef("Web Audio API") }}
+{{APIRef("Web Audio API")}}
 
-{{ domxref("AudioContext") }}的`createAnalyser()`方法能创建一个{{ domxref("AnalyserNode") }}，可以用来获取音频时间和频率数据，以及实现数据可视化。
+{{domxref("BaseAudioContext")}} 接口的 `createAnalyser()` 方法创建一个{{domxref("AnalyserNode")}}，它可以用来暴露音频时间和频率数据，以及创建数据可视化。
 
-> **备注：** 关于该节点的更多信息，请查看{{domxref("AnalyserNode")}}
+> **备注：** {{domxref("AnalyserNode.AnalyserNode", "AnalyserNode()")}} 构造函数是创建 {{domxref("AnalyserNode")}} 的推荐方法；请查看[创建 AudioNode](/zh-CN/docs/Web/API/AudioNode#creating_an_audionode)。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:11]:
```suggestion
> **备注：** {{domxref("AnalyserNode.AnalyserNode", "AnalyserNode()")}} 构造函数是创建 {{domxref("AnalyserNode")}} 的推荐方法；请查看[创建 AudioNode](/zh-CN/docs/Web/API/AudioNode#创建_audionode)。
```

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md [hunk 4]
```diff
@@ -1,78 +1,82 @@
...
 ### 返回值
 
-{{domxref("AnalyserNode")}}对象
+一个 {{domxref("AnalyserNode")}} 对象。
 
-## 例子
+## 示例
 
-下面的例子展示了 AudioContext 创建分析器节点的基本用法，然后用 requestAnimationFrame() 来反复获取时域数据，并绘制出当前音频输入的“示波器风格”输出。更多完整例子请查看[Voice-change-O-matic](https://mdn.github.io/voice-change-o-matic/) demo (中[app.js 的 128–205 行](https://github.com/mdn/voice-change-o-matic/blob/gh-pages/scripts/app.js#L128-L205)代码)
+下面的例子展示了 AudioContext 创建分析器节点的基本用法，然后用 requestAnimationFrame() 来反复获取时域数据，并绘制出当前音频输入的“示波器风格”输出。更多完整示例/信息请查看 [Voice-change-O-matic](https://mdn.github.io/voice-change-o-matic/) 实例（参阅 [app.js 的 108-193 行](https://github.com/mdn/webaudio-examples/tree/main/voice-change-o-matic/scripts/app.js#L108-L193)代码）。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:31]:
```suggestion
下面的示例展示了 AudioContext 创建分析器节点的基本用法，然后用 requestAnimationFrame() 来反复获取时域数据，并绘制出当前音频输入的“示波器风格”输出。更多完整示例/信息请查看 [Voice-change-O-matic](https://mdn.github.io/voice-change-o-matic/) 实例（参阅 [app.js 的 108-193 行](https://github.com/mdn/webaudio-examples/tree/main/voice-change-o-matic/scripts/app.js#L108-L193)代码）。
```

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md [hunk 5]
```diff
@@ -1,78 +1,82 @@
...
 
 analyser.fftSize = 2048;
-var bufferLength = analyser.fftSize;
-var dataArray = new Uint8Array(bufferLength);
+const bufferLength = analyser.frequencyBinCount;
+const dataArray = new Uint8Array(bufferLength);
 analyser.getByteTimeDomainData(dataArray);
 
-// draw an oscilloscope of the current audio source
+// 画出当前音频源的示波器
```
Comment (jasonren0403) [inline files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:44]:
```suggestion
// 绘制当前音频源的波形图
```

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md (hunk 2 above)
Comment (skyclouds2001) [reply files/zh-cn/web/api/baseaudiocontext/createanalyser/index.md:2]: 包裹标题的 `"` 也可以移除

**Review by jasonren0403 — APPROVED** [review]
Thanks @Yukiniro and @skyclouds2001 👍

**Pull request merged**